pkg/reconciler/buildrun/resources/taskrun.go

```diff
@@ -23,7 +23,7 @@ const (
 	inputSourceResourceName = "source"
```

Comment:
This file has only formatting changes. May you remove it from your PR?

Reply:
It's changed by `go fmt`, do we need to comply with this rule? Or you mean it should be address by a separate PR? Thanks.